### DISCUSSION TITLE
Create "canary statement"

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#411] [Feature Requests]
Create "canary statement" to let users know about goverment subpoena
E.g. https://riseup.net/en/about-us/canary

Moved from https://trello.com/c/brgfdHWx/58-create-canary-statement-to-let-users-know-about-goverment-subpoena

**Comment by Pineapplemint** [comment]
Hello, I'm new.
I came here to request this feature but I see it's already requested.
How do I add my support to this request? I'm clicking between Github and Trello but can't see anywhere to vote! Please count my comment as a vote?

**Reply by nguyenkims** [reply]
You can upvote the first post, we use this to prioritise feature developments.
